Assets/MRTK/Core/Interfaces/CameraSystem/IMixedRealityCameraSystem.cs

```diff
@@ -19,5 +19,12 @@ public interface IMixedRealityCameraSystem : IMixedRealityEventSystem, IMixedRea
         /// Is the current camera displaying on an opaque (VR / immersive) or a transparent (AR) device
         /// </summary>
         bool IsOpaque { get; }
+
+        /// <summary>
+        /// Override the camera's projection matrices for a smaller field of view
+        /// but rendered content will have more detail.  If holograms are not stable,
+        /// change the Stereo Rendering Mode from "Single Pass Instanced" to "Multi Pass."
```

Comment:
Nit: Would be probably good to have some <remarks> that only some platforms support this (i.e. I think just HL2 right?)

Reply:
Updated comment